packages/shared/src/env.ts

```diff
@@ -23,6 +23,8 @@ const EnvSchema = z.object({
       "ENCRYPTION_KEY must be 256 bits, 64 string characters in hex format, generate via: openssl rand -hex 32"
     )
     .optional(),
+  LANGFUSE_PROMPT_CACHE_ENABLED: z.enum(["true", "false"]).default("true"),
```

Comment:
id default to false here to make life easier for self hosters. We only enable this on AWS.